### DISCUSSION TITLE
Appimage: no right for inputs

### DISCUSSION THREAD

**Discussion by solisinvictum** [LizardByte/community#304] [Sunshine]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

User is in the input group. sunshine still complains:

[2023:09:15:06:14:31]: Error: Could not create Sunshine Mouse: Keine Berechtigung
[2023:09:15:06:14:31]: Error: Could not create Sunshine Touchscreen: Keine Berechtigung
[2023:09:15:06:14:31]: Error: Could not create Sunshine Keyboard: Keine Berechtigung
[2023:09:15:06:14:31]: Error: Unable to create some input devices! Are you a member of the 'input' group?

cat /etc/udev/rules.d/85-sunshine.rules 
KERNEL=="uinput", SUBSYSTEM=="misc", OPTIONS+="static_node=uinput", TAG+="uaccess"

### Host Operating System

Linux

### Operating System Version

Debian 12

### Architecture

64 bit

### Sunshine commit or version

latest

### Package

Linux - AppImage

### GPU Type

none (software encoding)

### GPU Model

123

### GPU Driver/Mesa Version

123

### Config

```Shell
123
```

### Relevant log output

```shell
123
```

**Comment by ReenigneArcher** [comment]
https://docs.lizardbyte.dev/projects/sunshine/en/latest/about/usage.html#linux

You need to restart the system after applying udev rules.

**Comment by solisinvictum** [comment]
I have of course.

Have now like mentioned on the other discussion, installed arch and compiled it from aur. this works.

On Debian 12 not.

[a comment by an automated account is not shown]
